### PR TITLE
OCPBUGS-9063: Remove frontend validation from legacy operand form gen…

### PR DESCRIPTION
**Pull request by TheRealJon**
…erator.

We rely on backend validation pretty much everywhere else in the repo, so brings the legacy form generator into alignment with convention.

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx [hunk 1]
```diff
@@ -504,7 +493,6 @@ const OperandFormInputGroup: React.FC<OperandFormInputGroupProps> = ({ error, fi
           {description}
         </span>
       )}
-      {error && <span className="co-error">{error}</span>}
```
[the remark this answers is not in the document]
Comment (TheRealJon) [reply frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx:507]: This is for frontend validation errors, which this PR removes.

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

👍

**Pull request merged**